ci/terraform/ticf-cri-stub/lambda.tf

```diff
@@ -7,7 +7,7 @@ module "ticf_cri_stub_role" {
 }
 
 module "ticf_cri_stub_lambda" {
-  source = "../modules/stub-endpoint-module"
+  source = "../modules/endpoint-lambda"
```

Comment:
needs to change?

Reply:
Yup - this new module is a rework of the old stub-endpoint-module (226873160032c33c8708b6d38d173d93af4d1917)